### PR TITLE
Feature/3452 score card examples

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Scorecard Examples and API docs page

### CODE REVIEW

**Comment by jeffvg** [comment]
is ```i blui-primary``` a thing? on score card with badge...

```<i blui-primary class="blui-grade_a"></i>```

**Comment by emclaug2** [comment]
> is `i blui-primary` a thing? on score card with badge...
> 
> `<i blui-primary class="blui-grade_a"></i>`

In this case the `blui-primary` selector isn't used on the scorecard, it's used on the Hero that's inside of the scorecard.

**Review by jeffvg — APPROVED** [review]
looks good, seems to be covering everything from before

File: src/docs/pages/component-docs/components/score-card/examples/with-score-badge.component.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+export const WITH_SCORE_BADGE = `<blui-score-card style="width: 350px" 
+    headerTitle="Substation 3"
+    headerSubtitle="High Humidity Alarm"
+    headerInfo="4 Devices"
+    [badgeOffset]="-54">
+    <mat-list blui-body>
+        <mat-list-item>Body Content</mat-list-item>
+    </mat-list>
+    <blui-hero blui-badge label="Grade" value="98" units="/100" [iconSize]="72">
```
Comment (huayunh) [inline src/docs/pages/component-docs/components/score-card/examples/with-score-badge.component.ts:11]: set "unitSpace" to "hide"

File: src/docs/pages/component-docs/components/score-card/examples/with-actions.component.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { Component } from '@angular/core';
+
+export const WITH_ACTIONS = `<blui-score-card style="width: 350px" class="with-actions" headerTitle="Header Actions">
+   <ng-container blui-action-items>
+        <mat-icon>search</mat-icon>
+        <mat-icon>mail</mat-icon>
+        <mat-icon>notifications</mat-icon>
```
Comment (huayunh) [inline src/docs/pages/component-docs/components/score-card/examples/with-actions.component.ts:7]: You typically see these icons here instead:

```suggestion
        <mat-icon>star</mat-icon>
        <mat-icon>settings</mat-icon>
        <mat-icon>more_vert</mat-icon>
```

**Review by huayunh — APPROVED** [review]
Looks good otherwise

File: src/docs/pages/component-docs/components/score-card/examples/with-score-badge.component.ts (hunk 1 above)
Comment (emclaug2) [reply src/docs/pages/component-docs/components/score-card/examples/with-score-badge.component.ts]: You got it

**Pull request merged**